### DISCUSSION TITLE
TDARR with Update Command

### DISCUSSION THREAD

**Discussion by xsasx** [tteck/Proxmox#3617] [Help Needed] [answered]
Hi, my Tdarr server has a newer version than my LXC Node. 

Sadly with update command it dont grap the actually version ? and if they dont have the same version they wont work together.

Any tipp how to update tdarr lxc manually ?

**Comment by Narkowii** [comment, marked as answer]
Hello
in Tdarr go to the OPTION menu and check the box Auto update non-Docker nodes
then reboot all your Tdarr LXC and it will auto update
this works for my 3 Tdarr LXC

**Reply by xsasx** [reply]
perfect ! that worked - thank you !
